kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.testing.collection;
+import java.util.*;
+import java.lang.*;
+
+public class OddNumbersExterminator {
+    ArrayList<Integer> even;
```

Comment:
tworzenie even jako zmienna globalna jest bledem.
Jak wywolasz 2x metode exterminate, to w even beda elementy z pierwszego wywolania i z drugiego.
Jak 3x, to z 1szego, 2giego i 3ciego. Niech even bedzie zmienna lokalna w metodzie exterminate